src/ert/scheduler/realization_state.py

```diff
@@ -117,7 +118,16 @@ def on_enter_state(self, target: RealizationState) -> None:
             asyncio.create_task(self.jobqueue._statechanges_to_publish.put(change))
 
     def on_enter_SUBMITTED(self) -> None:
-        asyncio.create_task(self.jobqueue.driver.submit(self))
+        task = asyncio.create_task(self._submit_async())
+        self.background_tasks.add(task)
+        task.add_done_callback(self.background_tasks.discard)
+
+    async def _submit_async(self) -> None:
+        try:
+            await self.jobqueue.driver.submit(self)
+        except Exception:
+            self.somethingwentwrong()
```

Comment:
Do we need `somethingwentwrong` when we do a `raise`? The `raise` should crash ert I suppose.